src/main/java/com/burgerking/Menu.java

```diff
@@ -0,0 +1,283 @@
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public void setName(String name) {
+        this.name = name;
+    }
+
+    public List<Food> getProductList() {
+        return productList;
+    }
+
+    public void setProductList(List<Food> productList) {
+        this.productList = productList;
+    }
+
+    public List<Food> getFilteredList() {
+        return filteredList;
+    }
+
+    public void setFilteredList(List<Food> filteredList) {
+        this.filteredList = filteredList;
+    }
+
+    public Map<FoodType, Integer> getAmountTypeMap() {
+        return amountTypeMap;
+    }
+
+    public void setAmountTypeMap(Map<FoodType, Integer> amountPerType) {
+        this.amountTypeMap = amountPerType;
+    }
+
+    public void addProduct(Food food){
+        this.productList.add(food);
+    }
+
+    public void removeProduct(Food food){
+        this.productList.remove(food);
+    }
+
+    private Comparator<Food> compareByPrice = new Comparator<Food>() {
+        @Override
+        public int compare(Food f1, Food f2) {
+            return f1.getPrice().compareTo(f2.getPrice());
+        }
+    };
+
+    private Comparator<Food> compareByName = new Comparator<Food>() {
+        @Override
+        public int compare(Food f1, Food f2) {
+            return f1.getName().compareTo(f2.getName());
+        }
+    };
+
+    public void sortByPrice(){
+        this.productList.sort(compareByPrice);
+    }
+
+    public void sortByName(){
+        this.productList.sort(compareByName);
+    }
+
+    public void display(List<Food> list){
+        int printIdCounter = 0;
+        for(Food food: list){
+            printIdCounter++;
+            System.out.format("%3s %8s %18s %7.2f$ %5d Cal. %8s",
+                    printIdCounter, food.getTypeName(), food.getName(), food.getPrice(), food.getCalories(), food.getSize());
+            System.out.println();
+        }
+        System.out.println();
+    }
+
+    public void filter(int filterOption) {
+        List<Food> filtered = new ArrayList<>();
+        for(Food food: this.getProductList()){
+            if(food.getType().ordinal()==filterOption){
+                filtered.add(food);
+            }
+        }
+        this.setFilteredList(filtered);
+    }
+
+    public boolean isOneOfBigger(Double price) {
+        for (Food food : this.getProductList()) {
+            if (food.getPrice() > price) {
+                return true;
+            }
+        }
+        return false;
+    }
+
+    public boolean isOneOfLower(Double price) {
+        for (Food food : this.getProductList()) {
+            if (food.getPrice() < price) {
+                return true;
+            }
+        }
+        return false;
+    }
+
+
+    public boolean isOneOfBigger(Integer calories){
+        for(Food food: this.getProductList()){
+            if(food.getCalories()>calories){
+                return true;
+            }
+        }
+        return false;
+    }
+
```

Comment:
`isOneOfBigger` method name is unclear. It is very difficult to understand what is method doing. Java convention says method name should represent logic of method body. This remark concern to most of methods in this class.

Reply:
change names of check by criteria and filter() and display() methods to more clear one